docs/docs.json

```diff
@@ -402,6 +402,7 @@
           {
             "group": "Client Libraries",
             "pages": [
+                  "reference/migration-guide",
```

Comment:
don't know where but we should find a better place. The reference documentation doesn't feel right. It might also be confusing for new users being there so prominent even before the library itself.